src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/NotifyOnVacancyActionsEventHandler.cs

```diff
@@ -36,10 +37,33 @@ public async Task Handle(VacancyClosedEvent notification, CancellationToken canc
                     await _vacancyStatusNotifier.VacancyManuallyClosed(vacancy);
                 }
             }
-            catch(NotificationException ex)
+            catch (NotificationException ex)
             {
                 _logger.LogError(ex, $"Unable to send notification for {nameof(VacancyClosedEvent)} and VacancyReference: {{vacancyReference}}", notification.VacancyReference);
             }
         }
+
+        public async Task Handle(LiveVacancyUpdatedEvent notification, CancellationToken cancellationToken)
+        {
+            try
+            {
+                var vacancy = await _vacancyRepository.GetVacancyAsync(notification.VacancyId);
+
+                switch (notification.UpdateKind)
```

Comment:
Would this be simpler as an `if`?

Reply:
This is subjective.

I thought about an `if` when writing this. I opted for the `switch` with fall-through cases where applicable because it is more readable to me in a collapsed form rather than an `if` in one long line or spanning two lines or multiple `if`'s.

The idea of the switch was also to cater for the not implemented case where an update kind could be `WageAdjustment` and the combination of that with a date change.